### ISSUE TITLE
How to Remove Album and Volume in Mini Player

### ISSUE THREAD

**Issue by bkribbs15** [skuethe/MMM-Spotify#145]
Hi, What are the options that need to be set in the config file in order to not show the Volume or Album in the Mini Player.

I see the example screenshot with Californication and I want to match that.

Thanks.

**Comment by skuethe** [comment]
[There are none](https://github.com/skuethe/MMM-Spotify#detail--default). Seems to be an old screenshot. But that should be easily implemented.
Send an PR if you like, otherwise I would need to see when I have time to do that.

**Comment by skuethe** [comment]
@bkribbs15 I've added a feature branch:
https://github.com/skuethe/MMM-Spotify/tree/feature/enhancement-for-145

please test and see if that works for you.

```js
  config: {
    [...]
    showAlbumLabel: true, // if you want to show the label for the current song album
    showVolumeLabel: true, // if you want to show the label for the current volume
    [...]
  }
```

**Comment by bkribbs15** [comment]
LGTM, ran just fine - Thanks for the quick update

**Comment by skuethe** [comment]
Okay, I'll merge it into development. I have some other things open right now, which I want to look at first before merging into master. I will link the PR to this issue to let you know when it is available / done.

**Comment by bkribbs15** [comment]
Sounds good, Thank you.

**Comment by skuethe** [comment]
Changes are now merged into master and available as [version `2.0.1`](https://github.com/skuethe/MMM-Spotify/releases/tag/v2.0.1)
